docgen/src/guide/Conditional_display.md

```diff
@@ -119,13 +119,11 @@ const IndexResults = connectStateResults(
 );
 
 const AllResults = connectStateResults(({ allSearchResults, children }) => {
-  const noResults =
+  const hasResults =
     allSearchResults &&
-    Object.values(allSearchResults).reduce(
-      (acc, results) => results.nbHits === 0,
-      false
-    );
-  return noResults ? (
+    Object.values(allSearchResults).some(results => results.nbHits !== 0);
```

Comment:
Would've written `> 0` personally, seems clearer

Reply:
Yep makes more sense the intent is a bit clearer 👍